components/camel-zeebe/src/main/docs/zeebe-component.adoc

```diff
@@ -0,0 +1,293 @@
+= Zeebe Component
+:doctitle: Zeebe
+:shortname: zeebe
+:artifactid: camel-zeebe
+:description: Zeebe component which does integrage with Camunda Zeebe to interact with the API.
+:since: 1.0-SNAPSHOT
+:supportlevel: Experimental
+:component-header: Both producer and consumer are supported
+
+*Since Camel {since}*
+
+*{component-header}*
+
+The *Zeebe*: components provides the ability to interact with business processes in https://github.com/camunda/zeebe[Zeebe].
+
+In order to use the Zeebe component, Maven users will need to add the
+following dependency to their `pom.xml`:
+
+Prerequisites
+
+You must have access to a local zeebe instance. More information is available at
+https://camunda.com/platform/zeebe/[Camunda Zeebe].
+
+*pom.xml*
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>camel-zeebe</artifactId>
+    <version>x.x.x</version>
+    <!-- use the same version as your Camel core version -->
+</dependency>
+----
```

Comment:
This seems a bit out of order. The dependency is listed after the prerequisites.

Reply:
It seems there are multiple flavors how adocs are structured. I changed it so that the pom info is at the end as in the redis component.

Reply:
Thanks. That works.